lib/yoti/dynamic_share_service/extension/thirdparty_attribute_extension.rb

```diff
@@ -13,10 +13,8 @@ def with_expiry_date(expiry_date)
         self
       end
 
-      def with_definitions(*names)
-        names.each do |s|
-          @definitions += [{ name: s }]
-        end
+      def with_definitions(*definitions)
+        @definitions += definitions
```

Comment:
This should probably create an `AttributeDefinition` for each string - e.g. https://github.com/getyoti/yoti-node-sdk/blob/master/src/dynamic_sharing_service/extension/third.party.attribute.extension.builder.js#L29